app/scripts/modules/kubernetes/src/v2/manifest/rollout/RollingRestart.tsx

```diff
@@ -0,0 +1,55 @@
+import { module } from 'angular';
+
+import * as React from 'react';
+import { react2angular } from 'react2angular';
+
+import { Application, ManifestWriter, ReactInjector } from '@spinnaker/core';
+
+import { IKubernetesServerGroupManager } from 'kubernetes/v2/serverGroupManager';
+
+interface IRollingRestartProps {
+  application: Application;
+  serverGroupManager: IKubernetesServerGroupManager;
+}
+
+interface IRollingRestartParameters {
+  account: string;
+  cloudProvider: string;
+  location: string;
+  manifestName: string;
+}
+
+function RollingRestart({ application, serverGroupManager }: IRollingRestartProps) {
+  function rollingRestart() {
+    const rollingRestartParameters: IRollingRestartParameters = {
+      account: serverGroupManager.account,
+      cloudProvider: 'kubernetes',
+      location: serverGroupManager.namespace,
+      manifestName: serverGroupManager.name,
+    };
+    ReactInjector.confirmationModalService.confirm({
+      account: serverGroupManager.account,
+      askForReason: true,
+      header: `Initiate rolling restart of ${serverGroupManager.name}`,
```

Comment:
"Initiate", 👍

Reply:
Haha yes thank you for workshopping the verbiage :pray: